examples/usecases/retrieval-with-hyperparameter-optimization.ipynb

```diff
@@ -0,0 +1,1405 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    train_set_transformed = workflow.fit_transform(train_set)
</pre><p>I'd shorten the names of the processed data sets like <code>train_transformed</code> <code>valid_transformed</code> but not a big deal.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/786/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
renamed